Logstash/logstash-input-azureblob/lib/logstash/inputs/azureblob.rb

```diff
@@ -265,7 +265,7 @@ def list_all_blobs
     @blob_list_page_size = 100 if @blob_list_page_size <= 0
     loop do
       # Need to limit the returned number of the returned entries to avoid out of memory exception.
-      entries = @azure_blob.list_blobs(@container, { :timeout => 10, :marker => continuation_token, :max_results => @blob_list_page_size })
+      entries = @azure_blob.list_blobs(@container, { :timeout => 60, :marker => continuation_token, :max_results => @blob_list_page_size })
```

Comment:
I recommend replacing the number with a constant

Reply:
@clguimanMSFT,
I am thinking replacing them either with constants or make them user configuration. Postpone the change a bit.